### ISSUE TITLE
Jetbrains Webstorm Find in Files floating window closing instantly on opening in tiling mode

### ISSUE THREAD

**Issue by Coniface** [pop-os/shell#1322]
I'd like to report a new floating window exception for the system.

**(1) Issue/Bug Description:**
Since Webstorm 2021.3.1 update I cannot open the "Find in files" window anymore in tiled mode. When I try to open it (Ctrl+Shift+F or via the menu), it instantly closes down. If I first disable tiling mode on PopOS (Super+Y) before opening it, it works perfectly.

**(2) Steps to reproduce (if you know):**
1. Install Webstorm 2021.3.1
2. Ensure tiling mode is on (Super+Y)
3. Try opening Find in Files dialog in Webstorm (Ctrl+Shift+F)
4. The dialog will open/close instantly on the display

**(3) Expected behavior:**
The Find in Files dialog should remain open.

**(4) Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="21.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.10"
VERSION_ID="21.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=impish
UBUNTU_CODENAME=impish
LOGO=distributor-logo-pop-os
```

**(5) Gnome Shell version:**
```
gnome-shell:
  Installed: 40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Candidate: 40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Version table:
 *** 40.5-1ubuntu2pop0~1633630918~21.10~91d3049 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
        100 /var/lib/dpkg/status
     40.5-1ubuntu2 500
        500 http://us.archive.ubuntu.com/ubuntu impish/main amd64 Packages
```

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**
```
pop-shell:
  Installed: 1.1.0~1641595320~21.10~c3d93f8
  Candidate: 1.1.0~1642015119~21.10~b3b2e35
  Version table:
     1.1.0~1642015119~21.10~b3b2e35 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
       1001 http://apt.pop-os.org/release impish/main i386 Packages
 *** 1.1.0~1641595320~21.10~c3d93f8 100
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**
Native

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
1 x 1080p

**(9) Other Installed/Enabled Extensions:**

**(10) Other Notes:**
This might be a common bug with the latest update of all JetBrains applications?

I tried to add it myself with the select tool but the window does not appear in the workspaces view (as I suppose it is a **dialog** and not a **window**?).

I tried using `xwininfo` to get the dialog title and manually add it to `~/.config/pop-shell/config.json` file but the dialog has no title (to the right of Window it, between quotes).
```
xwininfo: Window id: 0x22006af " "

  Absolute upper-left X:  429
  Absolute upper-left Y:  156
  Relative upper-left X:  429
  Relative upper-left Y:  156
  Width: 1313
  Height: 862
  Depth: 24
  Visual: 0x21
  Visual Class: TrueColor
  Border width: 0
  Class: InputOutput
  Colormap: 0x20 (installed)
  Bit Gravity State: NorthWestGravity
  Window Gravity State: NorthWestGravity
  Backing Store State: NotUseful
  Save Under State: no
  Map State: IsViewable
  Override Redirect State: no
  Corners:  +429+156  -178+156  -178-62  +429-62
  -geometry 1313x862+429-62
```

Here is what I pushed into the "float" array in the `config.json` file:
```
    {
      "class": "jetbrains-webstorm",
      "title": " "
    }
```

**Comment by SER-3604** [comment]
You're not running the version of pop-shell with the most recent commit.

Maybe try `sudo apt update`?

**Comment by Coniface** [comment]
Same problem after upgrading to
```
pop-shell:
  Installed: 1.1.0~1642015119~21.10~b3b2e35
  Candidate: 1.1.0~1642015119~21.10~b3b2e35
  Version table:
 *** 1.1.0~1642015119~21.10~b3b2e35 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
       1001 http://apt.pop-os.org/release impish/main i386 Packages
        100 /var/lib/dpkg/status
```

**Comment by Coniface** [comment]
No worries, yes I restarted several times.

**Comment by SER-3604** [comment]
I guess the latest commit might have not fixed cases outside of #1312.

**Comment by Coniface** [comment]
So sorry @SER-3604 I just understood your question.

> Not to state the obvious, but did you restart?

I had restarted multiple times since I first encountered the issue but I hadn't restarted after upgrading `pop-shell`!  
I just did it and it seems to work just fine. I'll close the issue for now, thank you for your help!
